sycl/test-e2e/Graph/Explicit/buffer_fill.cpp

```diff
@@ -0,0 +1,13 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+// Extra run to check for leaks in Level Zero using UR_L0_LEAKS_DEBUG
+// RUN: %if level_zero %{env UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck %s %}
```

Comment:
```suggestion
// RUN: %if level_zero %{env UR_L0_LEAKS_DEBUG=1 %{run} %t.out 2>&1 | FileCheck --implicit-check-not=LEAK %s %}
```

and drop line 6. Same in other file(s).

Reply:
It seems that https://github.com/intel/llvm/pull/12279 will update existing tests to the spelling I've suggested. Up to the graph devs on what the format for newly added tests should be.

Reply:
Updated this PR to use `--implicit-check-not=LEAK`, since that should be the format going forward.